tests/unit/multi.tcl

```diff
@@ -530,7 +530,7 @@ start_server {tags {"multi external-ok"}} {
         # releasing OOM
         $r2 config set maxmemory 0
         $r2 close
-    }
+    } {0} {needs:config-maxmemory}
```

Comment:
Why? I don't really see a problem with multiple tags.

Reply:
the question is how you decided these specific configs are the ones who cause issues, and not others, and what if someone wanna run it against a target that doesn't support CONFIG SET at all (like one that doesn't support DEBUG at all).
so maybe if allow wildcards, it gives us the ability to both control individual configs, but also disable all at once.

i don't want to write some script that run the test with skip tags for each of the ones we listed, and as soon as we add a new one (which i imagine we will, quite a lot), that script must be modified.

Reply:
I'm trying to avoid complexity and not try solve problems that don't yet exist. We could say that every test needs to include a definition of all the administrative commands it uses and we can filter tests by that - but I consider that over engineering.

Reply:
Ok, we can always improve later.